javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,109 @@
+//1
+const script = document.querySelector("script");
+script.addEventListener("load", function () { setTimeout(function () { console.log("Called after 2.5 seconds") }, 2500) });
+
+//2
+function loadedAfterxSeconds(delayInSeconds, stringToLog) {
+    setTimeout(function () { console.log(stringToLog) }, delayInSeconds * 1000);
+}
+
+loadedAfterxSeconds(3, "This string is logged after 3 seconds");
+loadedAfterxSeconds(5, "This string is logged after 5 seconds");
+
+function printClick() { console.log("click"); }//this is only made for testing
+
+//3
+const button = document.querySelector("button");
+button.addEventListener("click", function () { loadedAfterxSeconds(5, "This string is logged after 5 seconds") });
+
+//4
+function earthLogger() {
+    console.log("Earth")
+}
+function saturnLogger() {
+    console.log("Saturn")
+}
+
+function planetLogFunction(a) {
+    a()
+}
+
+planetLogFunction(earthLogger);
+planetLogFunction(saturnLogger);
+
+//5
+const logLocationButton = document.querySelector("button:nth-child(2)");
+
+function geo(position) {
+    let locationP = document.createElement("p");
+    locationP.innerHTML = `This is is the latitude: ${position.coords.latitude}.<br>This is the longitude: ${position.coords.longitude}`
+    document.getElementById("map").append(locationP);
+}
+
+logLocationButton.addEventListener("click", function () { navigator.geolocation.getCurrentPosition(geo) });
+
+//6 Map
+
+    function createMap(){
+    var script = document.createElement('script');
```

Comment:
Make sure your code is indented conistently.
```suggestion
function createMap(){
    var script = document.createElement('script');
```